src/Compilers/CSharp/Test/CommandLine/CommandLineTests.cs

```diff
@@ -5697,7 +5698,7 @@ public void IOFailure_MovePdbTemp()
             MockCSharpCompiler csc = MakeTrackingCsc(tempFilePaths, "/nologo", "/debug", "/preferreduilang:en", sourcePath);
             csc.FileMove = (source, dest) =>
             {
-                if (dest.EndsWith(".pdb"))
+                if (dest.EndsWith(".pdb", StringComparison.OrdinalIgnoreCase))
```

Comment:
Sort of. Depends on what we need to do. Ideally we avoid comparing paths or file names. If we really need to we have to think about the particular scenario. For example, in scripting the host might need to check the extension of the #r directive argument. If it is ".nuget" it should load a nuget package. Otherwise an assembly. In that case we can say that it's insensitive on Linux as well. All file names that end with ".nuget" (insensitively) will be treated on all platforms as nuget packages.